test/resources/policy_generator/acm-hardening_subscription.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: ManagedClusterSetBinding
```

Comment:
is this needed? Not sure what it is for. The Application is using placementrule so this is not used. The policies are created in `policies ` ns. So if this is for policies, it should be in `policies` ns. Did I understand it correctly?

Reply:
You're right. It should be for the `policies` namespace.

Reply:
It is a bit confusing I think.  The Subscription uses `PlacementRule` but the `PolicySet` uses `Placement`.  So this resource is needed by the `PolicySet` which is deployed by the subscription.

Reply:
I see. The `kubectl apply` specifies `-n namespace` which is `-n policies`.